recipes/sipbuildtool/conanfile.py

```diff
@@ -0,0 +1,127 @@
+import os
+import sys
+
+from pathlib import Path
+from io import StringIO
+
+from jinja2 import Template
+
+from conan import ConanFile
+from conan.tools.env import VirtualRunEnv
+from conan.tools.files import files
+from conans.errors import ConanException
+
+
+class SipBuildTool(object):
+    def __init__(self, conanfile: ConanFile):
+        self.conanfile = conanfile
+        env = VirtualRunEnv(self.conanfile)
+        self.run_env = env.environment()
+        self.envvars = None
+        self.python_venv_interpreter = None
```

Comment:
Maybe this should just be a local variable in the `configure` function? That's the only place where it's used.